packages/@o3r/schematics/src/utility/loaders.ts

```diff
@@ -92,6 +92,18 @@ export function getProjectFromTree(tree: Tree, projectName?: string | null, proj
   return workspaceProject;
 }
 
+/**
+ * Returns the root directory of a project with the given name ( a relative path from the project root)
+ *
+ * @param tree Files tree
+ * @param projectName Name of the project inside the workspace
+ */
+export function getProjectRootDir(tree: Tree, projectName?: string | null) {
```

Comment:
Maybe it would be cleaner to actually fix `getProjectFromTree` to not throw in case of `!tree.exists('angular.json')` instead of adding this helper (that can to be reproduced to all the fields of the workspace object)